Tests fail when using react@0.14

Commit b27a7a0 updated the `peerDependencies` so `react@0.14` can be used, but as far as I can tell `react-art` won't work with it.

Updating the `devDependencies` to use `react@0.14` causes the tests to fail. Below is an error from the tests that I also get when trying to use `react-art` from my own app:

```
 Error: /home/thom/dev/react-art/src/ReactART.js: Cannot find module 'react/lib/emptyObject' from '/home/thom/dev/react-art/src'
```

I assume `react/lib/emptyObject` is no longer part of the react API?

--- a/package.json
+++ b/package.json
@@ -18,7 +18,8 @@
     "url": "https://github.com/reactjs/react-art.git"
   },
   "dependencies": {
-    "art": "^0.10.0"
+    "art": "^0.10.0",
+    "fbjs": "^0.2.0"
   },
   "peerDependencies": {
     "react": "^0.13.0 || ^0.14.0-a"
@@ -29,7 +30,7 @@
     "gulp-react": "^2.0.0",
     "gulp-shell": "^0.3.0",
     "jest-cli": "^0.2.1",
-    "react": "^0.13.0",
+    "react": "^0.14.0-a",
     "react-tools": "^0.13.0"
   },
   "scripts": {

--- a/src/ReactART.js
+++ b/src/ReactART.js
@@ -22,7 +22,7 @@ var ReactMultiChild = require('react/lib/ReactMultiChild');
 var ReactUpdates = require('react/lib/ReactUpdates');
 
 var assign = require('react/lib/Object.assign');
-var emptyObject = require('react/lib/emptyObject');
+var emptyObject = require('fbjs/lib/emptyObject');
 
 var pooledTransform = new Transform();
 

